components/Navigation.tsx

```diff
@@ -27,174 +25,303 @@ type subroutes = subroute[];
 interface navigationItem {
   title: string;
   path: string;
-  iconName?: IconName;
+  iconName?: string | null;
   subroutes?: subroutes | null;
 }
 
 const navigation: navigationItem[] = [
-  { title: 'Getting Started', path: '/', iconName: 'rocket' },
+  {
+    title: 'What is Avo',
+    path: '/',
+    iconName: 'home',
+    subroutes: [
+      { type: 'route', title: 'What is Avo?', path: '/' },
+      {
+        type: 'route',
+        title: 'FAQs',
+        path: '/faqs/yes-you-can-faq',
+        subroutes: [
+          {
+            type: 'route',
+            title: 'What can I do with Avo',
+            path: '/faqs/yes-you-can-faq',
+          },
+          {
+            type: 'route',
+            title: 'Inspector FAQ',
+            path: '/faqs/faq-inspector',
+          },
+        ],
+      },
+    ],
+  },
   {
     title: 'Your Avo Workspace',
     path: '/workspace',
-    iconName: 'home',
+    iconName: 'toolbox',
```

Comment:
I've updated the icons to new ones but they are a bit off now, need to be moved a few pixels down. Could you take a look @logason or @hugihlynsson ?

I haven't figured out the previous approach, I think we can add icons to `Icon/customIcons.ts` somehow and then use them with the old approach, but I switched to a direct usage of svgs in an <img/> tag instead.

Reply:
On it!

Reply:
Fixed in 2912470 ✨

Reply:
awesome!